kinder/pkg/cluster/manager/actions/kubeadm-init.go

```diff
@@ -229,13 +212,7 @@ func postInit(c *status.Cluster, wait time.Duration) error {
 	}
 
 	if len(c.Workers()) == 0 {
-		// TODO: Once kubeadm 1.23 is no longer supported remove the <1.24 handling.
-		// TODO: Remove only the "control-plane" taint for kubeadm >= 1.25.
-		// https://github.com/kubernetes/kubeadm/issues/2200
-		taints := []string{"node-role.kubernetes.io/control-plane-", "node-role.kubernetes.io/master-"}
-		if cp1.MustKubeadmVersion().LessThan(version.MustParseSemantic("v1.24.0")) {
-			taints = []string{"node-role.kubernetes.io/master-"}
-		}
+		taints := []string{"node-role.kubernetes.io/control-plane-"}
 		taintArgs := []string{"--kubeconfig=/etc/kubernetes/admin.conf", "taint", "nodes", "--all"}
 		taintArgs = append(taintArgs, taints...)
```

Comment:
no longer to be a slice, fine as is.
```suggestion
		taint := "node-role.kubernetes.io/control-plane-"
		taintArgs := []string{"--kubeconfig=/etc/kubernetes/admin.conf", "taint", "nodes", "--all"}
		taintArgs = append(taintArgs, taint)
```